Examples/test-suite/ocaml/Makefile.in

```diff
@@ -85,9 +86,24 @@ include $(srcdir)/../common.mk
 	$(setup)
 	$(run_testcase)
 
+swig.cmi:
+	env SWIG_LIB=$(SWIG_LIB_DIR) $(SWIGEXE) -ocaml -co swig.mli
+	$(OCC) -c swig.mli
+swig.cmo:
+	env SWIG_LIB=$(SWIG_LIB_DIR) $(SWIGEXE) -ocaml -co swig.ml
+	$(OCC) -c swig.ml
+swigp4.cmi:
+	env SWIG_LIB=$(SWIG_LIB_DIR) $(SWIGEXE) -ocaml -co swigp4.ml
+	$(OCC) -I $(OCAMLP4WHERE) -pp "camlp4o pa_extend.cmo q_MLast.cmo" -c swigp4.ml
+extra_obs: swig.cmi swig.cmo swigp4.cmi
```

Comment:
Also I don't think make clean will remove the swig.mli, swig.mli, and swgip4.ml files and whatever else invoking $(OCC) on these files.

Reply:
Ignore that, ocaml_clean from Examples/Makefile.in is invoked and that looks like it cleans up.